ci-operator/step-registry/cucushift/installer/check/gcp/user-labels/filestore-csi/cucushift-installer-check-gcp-user-labels-filestore-csi-commands.sh

```diff
@@ -0,0 +1,279 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+if [ ! -f "${SHARED_DIR}/user_tags_sa.json" ]; then
+  echo "$(date -u --rfc-3339=seconds) - ERROR: Failed to find the key file of the IAM service-account for userTags testing on GCP."
+  exit 1
+fi
+
+if [ -f "${SHARED_DIR}/kubeconfig" ] ; then
+    export KUBECONFIG=${SHARED_DIR}/kubeconfig
+fi
+
+if [ -f "${SHARED_DIR}/proxy-conf.sh" ] ; then
+    source "${SHARED_DIR}/proxy-conf.sh"
+fi
+
+INFRA_ID="$(oc get infrastructures.config.openshift.io cluster -o jsonpath='{.status.infrastructureName}')"
+REGION="$(oc get infrastructures.config.openshift.io cluster -o jsonpath='{.status.platformStatus.gcp.region}')"
+vpc_network_name="$(oc get -o jsonpath='{.status.infrastructureName}{"\n"}' infrastructure cluster)-network"
+
+if [ -z "${INFRA_ID}" ] || [ -z "${REGION}" ] || [ -z "${vpc_network_name}" ]; then
+  echo "$(date -u --rfc-3339=seconds) - ERROR: Failed to get cluster info."
+  exit 1
+fi 
+
+GOOGLE_PROJECT_ID="$(< ${CLUSTER_PROFILE_DIR}/openshift_gcp_project)"
+export GCP_SHARED_CREDENTIALS_FILE="${SHARED_DIR}/user_tags_sa.json"
+sa_email=$(jq -r .client_email ${GCP_SHARED_CREDENTIALS_FILE})
+if ! gcloud auth list | grep -E "\*\s+${sa_email}"
+then
+  gcloud auth activate-service-account --key-file="${GCP_SHARED_CREDENTIALS_FILE}"
+  gcloud config set project "${GOOGLE_PROJECT_ID}"
+fi
+
+expected_labels="{\"createdby\":\"installer-qe\",\"environment\":\"test\",\"kubernetes-io-cluster-${INFRA_ID}\":\"owned\",\"storage_gke_io_created-by\":\"filestore_csi_storage_gke_io\"}"
```

Comment:
In this testing, shall we respect `USER_TAGS` or `USER_LABELS`, but form the following codes, seem like none of them are used.

Reply:
Updated, thanks!